ios/snowflake/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,39 +1,33 @@
 {
   "images" : [
     {
-      "size" : "29x29",
       "idiom" : "iphone",
-      "filename" : "Icon-App-29x29@2x.png",
```

Comment:
All these `filename` key:value pairs in this file need to be maintained.  The images are used by Apple when you submit it.  If you do a `find . -name *.png` in the `ios` directory, you'll see these files.